<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" total="5" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Command line suite" duration-ms="106568" started-at="2016-09-06T04:51:44Z" finished-at="2016-09-06T04:53:30Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="106568" started-at="2016-09-06T04:51:44Z" finished-at="2016-09-06T04:53:30Z">
      <class name="TestBase">
        <test-method status="PASS" signature="setUp()[pri:0, instance:TestBase@947bc5]" name="setUp" is-config="true" duration-ms="53" started-at="2016-09-06T06:51:44Z" finished-at="2016-09-06T06:51:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:TestBase@947bc5]" name="tearDown" is-config="true" duration-ms="29" started-at="2016-09-06T06:53:31Z" finished-at="2016-09-06T06:53:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestBase -->
      <class name="TestTut">
        <test-method status="PASS" signature="BeforeClassTestTut()[pri:0, instance:TestTut@9db43b]" name="BeforeClassTestTut" is-config="true" duration-ms="0" started-at="2016-09-06T06:51:44Z" finished-at="2016-09-06T06:51:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeClassTestTut -->
        <test-method status="PASS" signature="checkSendingMail(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:TestTut@9db43b]" name="checkSendingMail" duration-ms="101232" started-at="2016-09-06T06:51:44Z" data-provider="dateForLog" finished-at="2016-09-06T06:53:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[johnxotsin@tut.by]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[johnxotsin1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[alexwite@tut.by]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[alexwite1]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkSendingMail -->
        <test-method status="PASS" signature="takeScreenShotOnFailure(org.testng.ITestResult)[pri:0, instance:TestTut@9db43b]" name="takeScreenShotOnFailure" is-config="true" duration-ms="0" started-at="2016-09-06T06:53:25Z" finished-at="2016-09-06T06:53:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=checkSendingMail status=SUCCESS method=TestTut.checkSendingMail(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:TestTut@9db43b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenShotOnFailure -->
        <test-method status="FAIL" signature="checkSendingMail(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:TestTut@9db43b]" name="checkSendingMail" duration-ms="952" started-at="2016-09-06T06:53:25Z" data-provider="dateForLog" finished-at="2016-09-06T06:53:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[johnxotsin2@tut.by]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[johnxotsin21]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[alexwite2@tut.by]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[alexwite21]]>
              </value>
            </param>
          </params>
          <exception class="com.sun.mail.smtp.SMTPSendFailedException">
            <message>
              <![CDATA[553 5.7.1 Sender address rejected: not owned by auth user.
]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.sun.mail.smtp.SMTPSendFailedException: 553 5.7.1 Sender address rejected: not owned by auth user.
;
  nested exception is:
	com.sun.mail.smtp.SMTPSenderFailedException: 553 5.7.1 Sender address rejected: not owned by auth user.

	at com.sun.mail.smtp.SMTPTransport.issueSendCommand(SMTPTransport.java:2108)
	at com.sun.mail.smtp.SMTPTransport.mailFrom(SMTPTransport.java:1609)
	at com.sun.mail.smtp.SMTPTransport.sendMessage(SMTPTransport.java:1117)
	at javax.mail.Transport.send0(Transport.java:195)
	at javax.mail.Transport.send(Transport.java:124)
	at Function.sendEmail(Function.java:38)
	at TestTut.checkSendingMail(TestTut.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: com.sun.mail.smtp.SMTPSenderFailedException: 553 5.7.1 Sender address rejected: not owned by auth user.

	at com.sun.mail.smtp.SMTPTransport.mailFrom(SMTPTransport.java:1616)
	... 39 more
]]>
            </full-stacktrace>
          </exception> <!-- com.sun.mail.smtp.SMTPSendFailedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkSendingMail -->
        <test-method status="PASS" signature="takeScreenShotOnFailure(org.testng.ITestResult)[pri:0, instance:TestTut@9db43b]" name="takeScreenShotOnFailure" is-config="true" duration-ms="387" started-at="2016-09-06T06:53:26Z" finished-at="2016-09-06T06:53:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=checkSendingMail status=FAILURE method=TestTut.checkSendingMail(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:TestTut@9db43b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenShotOnFailure -->
        <test-method status="FAIL" signature="checkSendingMail(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:TestTut@9db43b]" name="checkSendingMail" duration-ms="1034" started-at="2016-09-06T06:53:26Z" data-provider="dateForLog" finished-at="2016-09-06T06:53:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[johnxotsin3@tut.by]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[johnxotsin31]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[alexwite3@tut.by]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[alexwite31]]>
              </value>
            </param>
          </params>
          <exception class="com.sun.mail.smtp.SMTPSendFailedException">
            <message>
              <![CDATA[553 5.7.1 Sender address rejected: not owned by auth user.
]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.sun.mail.smtp.SMTPSendFailedException: 553 5.7.1 Sender address rejected: not owned by auth user.
;
  nested exception is:
	com.sun.mail.smtp.SMTPSenderFailedException: 553 5.7.1 Sender address rejected: not owned by auth user.

	at com.sun.mail.smtp.SMTPTransport.issueSendCommand(SMTPTransport.java:2108)
	at com.sun.mail.smtp.SMTPTransport.mailFrom(SMTPTransport.java:1609)
	at com.sun.mail.smtp.SMTPTransport.sendMessage(SMTPTransport.java:1117)
	at javax.mail.Transport.send0(Transport.java:195)
	at javax.mail.Transport.send(Transport.java:124)
	at Function.sendEmail(Function.java:38)
	at TestTut.checkSendingMail(TestTut.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: com.sun.mail.smtp.SMTPSenderFailedException: 553 5.7.1 Sender address rejected: not owned by auth user.

	at com.sun.mail.smtp.SMTPTransport.mailFrom(SMTPTransport.java:1616)
	... 39 more
]]>
            </full-stacktrace>
          </exception> <!-- com.sun.mail.smtp.SMTPSendFailedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkSendingMail -->
        <test-method status="PASS" signature="takeScreenShotOnFailure(org.testng.ITestResult)[pri:0, instance:TestTut@9db43b]" name="takeScreenShotOnFailure" is-config="true" duration-ms="378" started-at="2016-09-06T06:53:28Z" finished-at="2016-09-06T06:53:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=checkSendingMail status=FAILURE method=TestTut.checkSendingMail(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:TestTut@9db43b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenShotOnFailure -->
        <test-method status="FAIL" signature="checkSendingMail(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:TestTut@9db43b]" name="checkSendingMail" duration-ms="843" started-at="2016-09-06T06:53:28Z" data-provider="dateForLog" finished-at="2016-09-06T06:53:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[johnxotsin4@tut.by]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[johnxotsin41]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[alexwite4@tut.by]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[alexwite41]]>
              </value>
            </param>
          </params>
          <exception class="com.sun.mail.smtp.SMTPSendFailedException">
            <message>
              <![CDATA[553 5.7.1 Sender address rejected: not owned by auth user.
]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.sun.mail.smtp.SMTPSendFailedException: 553 5.7.1 Sender address rejected: not owned by auth user.
;
  nested exception is:
	com.sun.mail.smtp.SMTPSenderFailedException: 553 5.7.1 Sender address rejected: not owned by auth user.

	at com.sun.mail.smtp.SMTPTransport.issueSendCommand(SMTPTransport.java:2108)
	at com.sun.mail.smtp.SMTPTransport.mailFrom(SMTPTransport.java:1609)
	at com.sun.mail.smtp.SMTPTransport.sendMessage(SMTPTransport.java:1117)
	at javax.mail.Transport.send0(Transport.java:195)
	at javax.mail.Transport.send(Transport.java:124)
	at Function.sendEmail(Function.java:38)
	at TestTut.checkSendingMail(TestTut.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: com.sun.mail.smtp.SMTPSenderFailedException: 553 5.7.1 Sender address rejected: not owned by auth user.

	at com.sun.mail.smtp.SMTPTransport.mailFrom(SMTPTransport.java:1616)
	... 39 more
]]>
            </full-stacktrace>
          </exception> <!-- com.sun.mail.smtp.SMTPSendFailedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkSendingMail -->
        <test-method status="PASS" signature="takeScreenShotOnFailure(org.testng.ITestResult)[pri:0, instance:TestTut@9db43b]" name="takeScreenShotOnFailure" is-config="true" duration-ms="463" started-at="2016-09-06T06:53:29Z" finished-at="2016-09-06T06:53:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=checkSendingMail status=FAILURE method=TestTut.checkSendingMail(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:TestTut@9db43b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenShotOnFailure -->
        <test-method status="FAIL" signature="checkSendingMail(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:TestTut@9db43b]" name="checkSendingMail" duration-ms="867" started-at="2016-09-06T06:53:29Z" data-provider="dateForLog" finished-at="2016-09-06T06:53:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[johnxotsin5@tut.by]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[johnxotsin51]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[alexwite5@tut.by]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[alexwite51]]>
              </value>
            </param>
          </params>
          <exception class="com.sun.mail.smtp.SMTPSendFailedException">
            <message>
              <![CDATA[553 5.7.1 Sender address rejected: not owned by auth user.
]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.sun.mail.smtp.SMTPSendFailedException: 553 5.7.1 Sender address rejected: not owned by auth user.
;
  nested exception is:
	com.sun.mail.smtp.SMTPSenderFailedException: 553 5.7.1 Sender address rejected: not owned by auth user.

	at com.sun.mail.smtp.SMTPTransport.issueSendCommand(SMTPTransport.java:2108)
	at com.sun.mail.smtp.SMTPTransport.mailFrom(SMTPTransport.java:1609)
	at com.sun.mail.smtp.SMTPTransport.sendMessage(SMTPTransport.java:1117)
	at javax.mail.Transport.send0(Transport.java:195)
	at javax.mail.Transport.send(Transport.java:124)
	at Function.sendEmail(Function.java:38)
	at TestTut.checkSendingMail(TestTut.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: com.sun.mail.smtp.SMTPSenderFailedException: 553 5.7.1 Sender address rejected: not owned by auth user.

	at com.sun.mail.smtp.SMTPTransport.mailFrom(SMTPTransport.java:1616)
	... 39 more
]]>
            </full-stacktrace>
          </exception> <!-- com.sun.mail.smtp.SMTPSendFailedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkSendingMail -->
        <test-method status="PASS" signature="takeScreenShotOnFailure(org.testng.ITestResult)[pri:0, instance:TestTut@9db43b]" name="takeScreenShotOnFailure" is-config="true" duration-ms="384" started-at="2016-09-06T06:53:30Z" finished-at="2016-09-06T06:53:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=checkSendingMail status=FAILURE method=TestTut.checkSendingMail(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:TestTut@9db43b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenShotOnFailure -->
      </class> <!-- TestTut -->
    </test> <!-- Command line test -->
  </suite> <!-- Command line suite -->
</testng-results>
